lightpath/ui/gui.py

```diff
@@ -1,14 +1,18 @@
 """
 Full Application for Lightpath
 """
+from functools import partial
 import logging
 import threading
 import os.path
 
+import numpy as np
+from pcdsdevices.device_types import *  # noqa
```

Comment:
`*` imports are the devil, but we'll let it slide... though it's odd to have style rules that get dumped when it's convenient.

Reply:
Yeah. I wanted to avoid two giant lists imports and then accepted classes, this felt cleaner. You disagree?

Reply:
It's slightly messier, but I would prefer something like:
```
import pcdsdevices.device_types as dtypes
shown_types = [dtypes.Attenuator, dtypes.IPM, dtypes.XFLS, ...]
```
So it's clear where the classes in `shown_types` come from.

Reply:
The fact that you need to `# noqa` it is a nudge that it's making your code less clear

Reply:
Good point. Didn't think to shorten. Will change